CODE_OF_CONDUCT.md

```diff
@@ -1,76 +1,123 @@
-# Contributor Covenant Code of Conduct
+# Code of Conduct
 
-## Our Pledge
+The OpenJS Foundation and its member projects use [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as their code of conduct. The full text is included [below](#contributor-covenant-code-of-conduct) in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, sex characteristics, gender identity and expression,
-level of experience, education, socio-economic status, nationality, personal
-appearance, race, religion, or sexual identity and orientation.
+## Commitment
 
-## Our Standards
+All recipients of reports commit to:
 
-Examples of behavior that contributes to creating a positive environment
-include:
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
+- participate in the path for escalation as outlined in the section on Escalation when required.
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+## Report an issue in a project
 
-Examples of unacceptable behavior by participants include:
+1. To report an issue in one of the projects listed below, please use the method provided. The project itself is responsible for managing these reports.
+    * **AMP Project:** <code-of-conduct@amp.dev>
+    * **Appium:** email maintainers
+    * **Electron:** <coc@electronjs.org>
+    * **Express.js:** <express-coc@lists.openjsf.org>
+    * **Fastify:** <hello@matteocollina.com> or <tommydelved@gmail.com>
+    * **HospitalRun:** <hello@hospitalrun.io>
+    * **LoopBack** <tsc@loopback.io>
+    * **Node.js:** <report@nodejs.org>
+    * **Node-RED:** <team@nodered.org>
+    * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
+    * **Webhint:** <support@webhint.io>
+2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
```

Comment:
i think instead of a list, we should just say they all go to my email

Reply:
I am going to consolidate everything under a reporting an issue subheading to email you as said above. Otherwise, after the deletion, it would look like this: 

```
## Report an issue in a project

To report an issue in one of the projects please email <ljharb@gmail.com>.


## Report an issue in a space managed by LJHarb
email <ljharb@gmail.com> to report an issue. 
``` 
And that seems unnecessarily repetitive.

Reply:
agreed, a single section seems fine.